### PR TITLE
add organization profile readme

### PR DESCRIPTION
**Pull request by whhe**
Add an [organization profile readme](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile#adding-a-public-organization-profile-readme) like https://github.com/github/.github.

The rendered page can refer to https://github.com/oceanbase/.github/blob/profile-readme/profile/README.md

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#### Ecosystem Projects
+
+The OceanBase community provides many solutions together with other communities and partners, such as [CloudCanal](https://www.clougence.com/cloudcanal), [Flink CDC](https://github.com/ververica/flink-cdc-connectors) and [Apache DolphinScheduler](https://github.com/apache/dolphinscheduler) etc.
+
+We will create new repository to collect all the information about ecosystem projects on GitHub in the near future, and welcome more communities and developers to join our ecosystem. Let's do some cool stuff together!
+
+### Get in touch
+
+- GitHub: [Issues](https://github.com/oceanbase/oceanbase/issues) [Discussions](https://github.com/oceanbase/oceanbase/discussions)
+- Forum: https://ask.oceanbase.com
```
Comment (hnwyllmm) [inline profile/README.md]: 要不要写下简体中文

Comment (whhe) [reply profile/README.md]: > 要不要写下简体中文

可以

**Comment by whhe** [comment]
@longdafeng @caifeizhi PTAL

**Comment by longdafeng** [comment]
+1

**Comment by caifeizhi** [comment]
I suggest waiting until [ oceanbase/oceanbase readme](https://github.com/oceanbase/oceanbase/pull/1350)  is merged before reviewing yours, as these two PRs have similar content (like `key feature` and `get in touch `part)

**Review by caifeizhi — APPROVED** [review]
LGTM

**Review by caifeizhi — APPROVED** [review]
LGTM

**Pull request merged**